src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -108,6 +108,9 @@ const acdcChangeHandler = async (
   if (event.payload.status === CredentialStatus.PENDING) {
     dispatch(setCurrentOperation(OperationType.ADD_CREDENTIAL));
     dispatch(setToastMsg(ToastMsgType.CREDENTIAL_REQUEST_PENDING));
+  } else if (event.payload.status === CredentialStatus.REVOKED) {
+    dispatch(updateOrAddCredsCache(event.payload.credential));
+    dispatch(setCurrentOperation(OperationType.IDLE));
```

Comment:
I don't think we need to change the current operation since this is a background task and may interfere with what the user is currently doing

Can you test this out? Maybe we can also remove the current operation change in the `else` block below.

cc @sdisalvo-crd

Reply:
@sdisalvo-crd Hmm, so when we get a pending credential it will redirect to the credentials page because of the `OperationType.ADD_CREDENTIAL`. I guess before we assumed it would appear quickly and when confirmed we would set `OperationType.IDLE`. But we can't guarantee when it'll appear - so setting it to `IDLE` could cause conflicting issues if you are e.g. in the middle of scanning a different QR code.

Is there any danger in the `OperationType` staying as `ADD_CREDENTIAL`?

Reply:
As per our chat conversation, feel free to remove any legacy code that you feel is unnecessary.

Reply:
OK @Sotatek-TungNguyen2a here is the plan:
- Remove all `setCurrentOperation` calls in this function, including the one for `ADD_CREDENTIAL`
- Remove `ADD_CREDENTIAL` from the enum
- Remove the check for `ADD_CREDENTIAL` in nextRoute.ts

This is legacy code from before. Before you were able to receive credentials by scanning QR codes but this isn't possible in KERI right now.

Reply:
I am not sure about this. But I tested it and it still work as normal

Reply:
all good, I'm confident on it - once you have it pushed and its approved I will run locally to test myself too before merging